src/clock.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
File should be capital `Clock.js` as this is a component. Components always capitalized